e2e/pnpm_workspace/WORKSPACE

```diff
@@ -1,3 +1,18 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "aspect_bazel_lib",
+    sha256 = "262e3d6693cdc16dd43880785cdae13c64e6a3f63f75b1993c716295093d117f",
+    strip_prefix = "bazel-lib-1.38.1",
+    url = "https://github.com/aspect-build/bazel-lib/releases/download/v1.38.1/bazel-lib-v1.38.1.tar.gz",
+)
+
+load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies", "register_coreutils_toolchains")
+
+aspect_bazel_lib_dependencies()
+
+register_coreutils_toolchains()
```

Comment:
I shouldn't have to add this to the bazel-lib 1.x e2e, but it is needed for the bazel-lib 2.x version of the e2e. So flipping the version isn't always the only thing needed to test both. Perhaps this is a good enough reason not to go with this approach..

Reply:
I could add an input to the e2e copying macro that takes buildozer commands to apply to the WORKSPACE and the 2.0 version could be modified that way..